cvat/settings/base.py

```diff
@@ -285,7 +285,11 @@ def add_ssh_keys():
 # IAM settings
 IAM_TYPE = 'BASIC'
 IAM_BASE_EXCEPTION = None # a class which will be used by IAM to report errors
-IAM_DEFAULT_ROLES = ['user']
+
+# the function should be in uppercase to able get access from django.conf.settings
+def GET_IAM_DEFAULT_ROLES(user) -> list:
```

Comment:
Please add FIXME here. There are several ways to "replace" default IAM role. One of them is to assign groups when you create a user inside Crowdsourcing plugin and don't add more groups if user.groups field isn't empty. Need to check that.